src/views/SignUp.vue

```diff
@@ -1,18 +1,14 @@
 <template>
-  <fwb-alert type="success" v-if="showSuccessAlert">
-    Your form is submitted successfully!
-  </fwb-alert>
-  <fwb-input v-model="name" label="Name" />
-  <fwb-select v-model="selected" :options="league" label="League" />
-  <fwb-toast divide v-if="selected"> Sign up will end on {{ getEndDate(selected) }}. </fwb-toast>
-  <fwb-button gradient="red" @click="submitForm">Confirm</fwb-button>
+  <fwb-alert type="success" v-if="showSuccessAlert"> "Zostałeś zapisany!" </fwb-alert>
```

Comment:
Okay, remove the "" symbols. Also please trim it. We don't need additional whitespace.